packages/core/src/Accordion/Accordion.styled.tsx

```diff
@@ -23,6 +23,17 @@ const slideUp = keyframes`
   }
 `
 
+export const StyledAccordionRoot = styled(Accordion.Root)`
```

Comment:
```suggestion
export const StyledAccordionRoot = styled(Accordion.Root)<IAccordionProps>`
```

(or whatever the name of the accordion props interface is)

Reply:
Updated